case-lib/apause.exp

```diff
@@ -137,8 +137,20 @@ set volume_always_zero true
 
 expect {
 
+    # `man re_syntax` claims that Tcl regular expressions are compliant
+    # with the basic and extended POSIX ones while adding a 3rd,
+    # extended flavor.  It's not clear which flavor `expect -re` uses
+    # but it's not the basic one.
+    # Use {} not "" to avoid quoting issues and backslash proliferation.
+
     # When multiple patterns match, first pattern wins.
 
+    -nocase -re {error.*\r|PAUSE.*no[[:blank:]]*hw[[:blank:]]*support.*\r} {
```

Comment:
laughing at my desk at 8am, no idea what this does haha.

Reply:
It took me only a couple days of practice to become productive in Tcl and expect (without any "stackoverflow garbage in, AI garbage out!); it's not so bad.

`{ }` is like the single quotes in shell except they can be nested.

The basic syntax of the `expect` instruction is just:
```
expect {
 { pattern1 } { action1 }
 { pattern2 } { action2 }
 ...
}
```
`-nocase -re` means the pattern is a case-insensitive regular expression (as opposed to globbing).

That regular expression has itself nothing specific to Tcl.

Thanks for taking a look!